### PR TITLE
fix(build): ensure jest is able to run tests in all environments

### PR DESCRIPTION
**Pull request by austingreendev**
## Description

Ensure jest is able to finish test runs in all environments. Related to https://github.com/facebook/jest/issues/10777

## Checklist

- [ ] ~:ok_hand: design updates are Garden Designer approved (add the
      designer as a reviewer)~
- [ ] ~:globe_with_meridians: demo is up-to-date (`yarn start`)~
- [ ] ~:arrow_left: renders as expected with reversed (RTL) direction~
- [ ] ~:metal: renders as expected with Bedrock CSS (`?bedrock`)~
- [ ] ~:wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver~
- [ ] ~:guardsman: includes new unit tests~
- [ ] ~:memo: tested in Chrome, Firefox, Safari, Edge, and IE11~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -146,6 +146,7 @@
     "webpack": "4.44.1"
   },
   "resolutions": {
-    "@types/vfile-message": "1.0.1"
+    "@types/vfile-message": "1.0.1",
+    "**/jest-haste-map/fsevents": "2.2.1"
```
Comment (jzempel) [inline package.json:150]: This this comment worth considering? https://github.com/facebook/jest/issues/10777#issuecomment-722299866

**Review by jzempel — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (austingreendev) [reply package.json:150]: I tried to ensure that the new patch still make it through to support linux/windows, but I guess that didn't work. Let me give that a shot.

Comment (austingreendev) [reply package.json]: This should be fixed now. Had to mess with the lockfile a little more than the comments suggested.

Comment (jzempel) [reply package.json]: Even better. :pray:

**Pull request merged**